deeprank/features/Edesolv.py

```diff
@@ -0,0 +1,411 @@
+import warnings
+
+from pdb2sql import pdb2sql, interface
+from deeprank.features import FeatureClass
+from copy import deepcopy
+from io import StringIO
+import numpy as np
+
+try:
+    from Bio.PDB import PDBParser, SASA
+
+except ImportError:
+    warnings.warn('Biopython module not found. Make sure you have Biopython 1.79 or later')
+    raise
+
+
+class Edesolv(FeatureClass):
+
+    # init the class
```

Comment:
Please leave a comment about the equations and references used for calculating Edesolv.

Reply:
Do you mean in the class docstring?

Reply:
Yep

Reply:
Improved in 6fed0fc . Please let me know if it's not satisfactory yet.